types/src/binary_port/record_id.rs

```diff
@@ -0,0 +1,105 @@
+use core::convert::TryFrom;
+
+#[cfg(test)]
+use rand::Rng;
+use serde::Serialize;
+
+#[cfg(test)]
+use crate::testing::TestRng;
+
+/// An identifier of a record type.
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, Serialize)]
+#[repr(u16)]
+pub enum RecordId {
+    /// Refers to `BlockHeader` db.
```

Comment:
```suggestion
    /// Refers to `BlockHeader` record.
```

But not sure about this. "Record" refers to a single, well, record in a DB.